repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -396,7 +377,11 @@ public EntityMutationResponse mapAttributesAndClassifications(EntityMutationCont
                     reqContext.cache(createdEntity);
 
                     if (DEFERRED_ACTION_ENABLED) {
-                        deleteDelegate.getHandler().createAndQueueTask(CLASSIFICATION_ONLY_PROPAGATION_DELETE, reqContext.getDeletedEdgesIds());
+                        List<AtlasEdge> classificationEdges = getAllClassificationEdges(vertex);
```

Comment:
In case multiple edges are deleted having a couple of classifications, 2 tasks will be created 

I just wanted to check if that is intentional & we don't want to create a separate tasks for each edge, tag pair
I think reason for creating separate tasks per tag is to better management in terms of "retry" if needed

there are 2 approaches I can think of
1. create a single task, and have a couple of for loops to process each tag separately
2. create separate tasks for each pairs of edge & tag to better track & retry ability

Reply:
+1

Reply:
We want to fragment the task in case some task got preempted midway, it didn't have to perform all the traversal again.
My suggestion would be -->
      For each edge get the attached classification and create the task for each classification and edge. e.g 2 classification 1,2 attached to deleted edge A, then tasks would be [ 1,A],[2,A]
 
And I also have some confusion over the logic implemented here because for each classification there might be different traversal so may have to handle differently.

Reply:
The original approach was to separate execution into different tasks per classification. We can discuss this in detail as a team.

Reply:
Addressed